docs/ruleset_functions/is_baseline_system_1(HVAC_id).md

```diff
@@ -0,0 +1,52 @@
+# is_baseline_system_1  
+
+**Description:** Get either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 1 (PTAC), system 1a (system 1 with purchased CHW), system 1b (system 1 with purchased heating), system 1c (system 1 with purchased CHW and purchased HW).  
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled as either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 in the B_RMR.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_baseline_system_1**: The function returns either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 1 (PTAC), system 1a (system 1 with purchased CHW), system 1b (system 1 with purchased heating), system 1c (system 1 with purchased CHW and purchased HW).  
+ 
+**Function Call:** 
+1. get_hvac_zone_list_w_area()  
+2. is_heating_type_fluid_loop()
+3. is_cooling_type_DX()
+4. serves_single_zone()  
+5. is_fan_CV()  
+6. is_fluid_loop_attached_to_boiler()
+7. is_fluid_loop_purchased_heating()
+8. is_fluid_loop_purchased_CHW()
+9. is_cooling_type_fluid_loop()
+
+## Logic:    
+- Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
+- Check that there is no preheat system (it equals Null), if there is none then carry on: `if Len(hvac_b.preheat_system) == Null:`  
+    - Check if heatingsystem is a fluid_loop, if it is then carry on: `if is_heating_type_fluid_loop(B_RMR, hvac_b.id) == TRUE:`     
+        - Check if fansystem is constant volume, if yes then carry on: `if is_fan_CV(B_RMR, hvac_b.id) == TRUE:`  
+            - Check if the hvac system serves a single zone, if yes carry on: `if serves_single_zone(B_RMR, hvac_b.id) == TRUE:`   
+                - Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`  
+                - Get list of zones that the HVAC system serves (should only be one): `zone_list_b = hvac_zone_list_w_area_dict_b[hvac_b.id]["ZONE_LIST"]`  
+                - Create an object for the zone associated with the HVAC system: `zone_b = zone_list_b[0]`
```

Comment:
not sure why this is needed. Would it be sufficient to simply identify if the system serves a single zone (done by line 30)

Reply:
@supriyagoel We need to the zone to get the terminal unit which allows us to do our terminal unit checks.